android/src/main/java/com/wix/reactnativenotifications/core/notifications/OreoNotifications.java

```diff
@@ -0,0 +1,51 @@
+package com.wix.reactnativenotifications.core.notifications;
+
+import android.app.Notification;
+import android.annotation.TargetApi;
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.content.Context;
+import android.os.Build;
+import android.util.Log;
+
+import static com.wix.reactnativenotifications.Defs.LOGTAG;
+
+public class OreoNotifications {
+
+    public static void initialize(Context context) {
+        Log.d(LOGTAG, "OreoNotifications.initialize()");
+
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.O) return;
+
+        createDefaultChannel(context);
+    }
+
+    private static String getChannelName(Context context) {
+        return getChannelId(context);
+    }
+
+    public static String getChannelId(Context context) {
+        return context.getPackageName() + "_default";
+    }
+
+    public static void setChannel(Context context, Notification.Builder builder) {
+        Log.d(LOGTAG, "OreoNotifications.setChannel()");
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.O) return;
```

Comment:
As noted this comparison should be flipped < instead of >

Reply:
That's weird. I pushed the fix to this branch and merged to master this arvo. The PR diff must become immutable after merge.

TL;DR Fixed and merged.